components/brave_rewards/resources/shared/lib/provider_payout_status.ts

```diff
@@ -11,5 +11,5 @@ export function getProviderPayoutStatus (
   walletProvider: ExternalWalletProvider | null
 ): ProviderPayoutStatus {
   const key = walletProvider || 'unverified'
-  return payoutStatus[key] || 'off'
+  return payoutStatus ? payoutStatus[key] || 'off' : 'off'
```

Comment:
If `payoutStatus` is coming in as undefined, we must have a type violation somewhere. I'm thinking we need to fix `RewardsDOMHandler::OnGetRewardsParameters` (in `brave_rewards_page_ui.cc`) to correctly handle the case where `parameters` is null.

Reply:
Fixed in 36415a0017ab2d39c667cedf82fbced38ecfd8bb